.github/pull_request_template.md

```diff
@@ -18,3 +18,10 @@ Please tick as appropriate and edit the reasons (e.g.: "backport: not needed bec
 
 Please refer to the [contributing guidelines](https://github.com/Mbed-TLS/mbedtls/blob/development/CONTRIBUTING.md), especially the
 checklist for PR contributors.
+
+Help make review efficient:
+* Multiple simple commits
+  - please structure your PR into a series of small commits, each of which does one thing
+* No force-push
+  - please do not force-push to update your PR - just add new commit(s)
+  - use `git merge` (or the GitHub web interface) to resolve conflicts - not `git rebase`
```

Comment:
Another annoying thing external contributors sometimes to is to merge even when there are no conflicts. I guess they're influenced by some projects that want pull requests to always be a fast-forward from the target branch. I don't know if it's prevalent enough that we need to explicitly admonish against it, but if we mention merging, we should make it clear that we only want merging if it's really necessary.

Reply:
Removed this line - we can cover this in more detail in RTD.